genesis/ext/pyrender/numba_gl_wrapper.py

```diff
@@ -49,11 +66,27 @@ def __init__(self):
 
     def load_func(self, func_name, *signature):
         dll = GL.platform.PLATFORM.GL
-        func_ptr = GL.platform.ctypesloader.buildFunction(
-            GL.platform.PLATFORM.functionTypeFor(dll)(*signature),
-            func_name,
-            dll,
-        )
+        func_type = GL.platform.PLATFORM.functionTypeFor(dll)(*signature)
+        try:
+            func_ptr = GL.platform.ctypesloader.buildFunction(
+                func_type,
+                func_name,
+                dll,
+            )
+        except:
+            func_type = func_type()
+            func_ptr = glplatform.createExtensionFunction(
```

Comment:
glplatform.crateExtensionFunction() must be GL.platform.createExtensionFunction()